flight/PiOS/Common/pios_mpu6000.c

```diff
@@ -178,6 +180,20 @@ int32_t PIOS_MPU6000_Init(uint32_t spi_id, uint32_t slave_num, const struct pios
 }
 
 /**
+ * @brief Updates the MPU6000 chip orientation.
```

Comment:
So I'm in favor of removing this orientation stuff for onboard parts, instead using the yaw/pitch/roll fields for the attitude / levelling instead.